static/templates/pod.html

```diff
@@ -0,0 +1,12 @@
+<div class="panel panel-default">
```

Comment:
Thanks, will change to `[[`s. I see the interpolation symbols have been [configured](https://github.com/rapidpro/casepro/blob/master/static/coffee/app.coffee#L15-L16) to use `[[`s. For some reason, angular still seems to accept `{{` for interpolation, but makes sense to stick to `[[`s for consistency anyways.